client/jetpack-connect/controller.js

```diff
@@ -213,6 +214,13 @@ export function signupForm( context, next ) {
 	next();
 }
 
+export function credsForm( context ) {
+	removeSidebar( context );
```

Comment:
Thanks to Single-Tree Rendering (props Mikael), we might not need this anymore. Maybe try if it works without.

Reply:
right, I recall now 👍